test/studies/lcals/LongDouble.chpl

```diff
@@ -0,0 +1,39 @@
+/* A module to provide a "long double" type to Chapel that matches the
+   underlying C compiler's "long double" type.
+ */
+
+require "longdouble.c", "longdouble.h";
+
+extern type longdouble;
+
+extern proc longdoubleToDouble(d: longdouble): real(64);
+
+proc _cast(type t, ld: longdouble) where t == real {
+  return longdoubleToDouble(ld);
```

Comment:
Yes, a primitive cast will work here.  I didn't realize that.  I'll make this change.  While I'm at it, I can change the where clause to allow any size 'real' or 'int' as the target type.  Since that was the only function in "longdouble.c" I can remove it too.